src/accessibility.cs

```diff
@@ -42,7 +42,7 @@ interface AXCustomContent : NSCopying, NSSecureCoding
 	}
 
 	[Watch (7, 0), TV (14, 0), Mac (10, 16), iOS (14, 0)]
-	[Protocol, Model]
+	[Protocol]
 	[BaseType (typeof(NSObject))]
```

Comment:
you can't have a `[BaseType]` for a protocol (generator won't do the correct thing)
if needed, normal subclasses works for them